react/features/toolbox/components/native/OverflowMenu.js

```diff
@@ -128,6 +129,7 @@ class OverflowMenu extends PureComponent<Props, State> {
                 <InviteButton { ...buttonProps } />
                 <AudioOnlyButton { ...buttonProps } />
                 <RaiseHandButton { ...buttonProps } />
+                <LobbyModeButton { ...buttonProps } />
```

Comment:
Because I think this should be in a more prominent place. But this will also change with the security menu (that for example should also be here in the open list, not in the collapsed one)